.github/workflows/ci.yaml

```diff
@@ -95,8 +98,8 @@ jobs:
       if: ${{ needs.changes.outputs.non-docs == 'true' }}
       with:
         version: v2.1.6
-        only-new-issues: true
-        args: --timeout=10m
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is this now needed? 
From the docs:

```
github-token

(optional)

When using the only-new-issues option, the GitHub API is used, so a token is required.

By default, it uses the github.token from the action.
```

Since `only-new-issues` is removed, it's not needed. In any case `GITHUB_TOKEN` is the default value.